commonUI/src/main/res/values/colors.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="primaryColor">#D63447</color>
+    <color name="primaryColor">#48A145</color>
```

Comment:
lets remove this change from this commit, this should be a separate task in itself, this requires much more fixes as primary color is used at many places